### DISCUSSION TITLE
Add multiple record to many to many table

### DISCUSSION THREAD

**Discussion by aramis-it** [prisma/prisma#4779] [Q&A] [answered]
I tried following but not working
table1 < table1table2 > table2

```json
{
    "data": {
        "table1": {
            "connect": {
                "id": 2
            }
        },
        "table2": {
            "connect": [{
                "id": 1
            },
            {
                "id": 2
            }]
        }
    }
}
```

**Comment by aramis-it** [comment, marked as answer]
I got it
```json
{
    "data": {
        "title": "tit1",
        "table1table1": {
            "create": [
                {
                    "table1alias": {
                        "create": {
                            "title": "t11"
                        }
                    }
                },
                {
                    "table1alias": {
                        "create": {
                            "title": "t12"
                        }
                    }
                }
            ]
        }
    }
}
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
